generator/C/test/posix/testmav.c

```diff
@@ -120,6 +120,9 @@ static void print_message(mavlink_message_t *msg)
 	}
 	const mavlink_field_info_t *f = m->fields;
 	unsigned i;
+	printf("sysid:%d ", msg->sysid);
```

Comment:
Is this stray debug?

Reply:
Its a deliberate addition to the C tests/binding/s that causes them to emit enough human-readable info about each of the 'test' packets so that a human can cross-reference what the C bindings think the packet is with what the testparser.js thinks the bytes mean. future todo - at some point it would be nice to  extend testparser.js to also consume this data and cross-check the internal/s of the packets as well, not just 'did the hex parse into a known packet type' that it does now.